config/checkstyle-input-suppressions.xml

```diff
@@ -16,11 +20,13 @@
   <suppress id="noCheckstyleInInputs"
           files="[\\/]checks[\\/]imports[\\/]unusedimports[\\/]InputUnusedImportsAnnotations.java"/>
   <suppress id="noCheckstyleInInputs"
-          files="[\\/]checks[\\/]coding[\\/]illegaltype[\\/]InputIllegalTypeSameFileNameFalsePositive.java"/>
+          files="[\\/]checks[\\/]imports[\\/]avoidstarimport[\\/]InputAvoidStarImportAllowClass.java"/>
+  <suppress id="noCheckstyleInInputs"
+          files="[\\/]checks[\\/]imports[\\/]avoidstarimport[\\/]InputAvoidStarImportAllowStaticMember.java"/>
```

Comment:
Why are we adding suppressions for `avoidstarimport` when this PR is just for `illegaltype`?

Reply:
done

Reply:
@suniti0804 Please be careful and review when you are resolving conflicts during a rebase. It looks like this was leftover from someone else's commit during the middle of your work here. This would have brought back suppressions that were already resolved.